solana-programs/anchor/audius-data/programs/audius-data/src/lib.rs

```diff
@@ -369,6 +438,10 @@ pub const TRACK_ACCOUNT_SIZE: usize = 8 + 32 + 8;
 /// 8 bytes (anchor prefix) + 32 (PublicKey) + 8 (id)
 pub const PLAYLIST_ACCOUNT_SIZE: usize = 8 + 32 + 8;
 
+/// Size of user authority delegation account
+/// 8 bytes (anchor prefix) + 32 (PublicKey) + 8 (id)
```

Comment:
should update this comment and maybe specify that it's the user auth and delegate.

Reply:
yes great callout